src/db/helpers/influx.py

```diff
@@ -0,0 +1,69 @@
+""" wrapper around time series database"""
+
+import os
+from dotenv import load_dotenv
+from src.utils.logger import logging as logger
+from influxdb_client_3 import InfluxDBClient3
+
+load_dotenv()
+
+INFLUX_TOKEN = str(os.getenv("INFLUX_TOKEN"))
+INFLUX_HOST = str(os.getenv("INFLUX_HOST"))
+INFLUX_DB = str(os.getenv("INFLUX_DB"))
+
+
+class InfluxDB:
+    """ helper classes for influx time-series database"""
+
+    def __init__(self) -> None:
+        """connects to the influx time-series database"""
+        try:
+            logger.info(f"connecting to influx db at {INFLUX_HOST} and DB {INFLUX_DB}")
+            self.client = InfluxDBClient3(token=INFLUX_TOKEN,
+                                          host=INFLUX_HOST,
+                                          database=INFLUX_DB)
+        except ConnectionError as error:
+            logger.error(
+                f"error connecting to quadrant db at {INFLUX_HOST} and DB {INFLUX_DB}"
```

Comment:
Souldnt  this read as f"error connecting to influx  db at {INFLUX_HOST} and DB {INFLUX_DB}"